packages/core/src/utils.js

```diff
@@ -220,10 +220,22 @@ function computeDefaults(
     );
   } else if ("oneOf" in schema) {
     schema =
-      schema.oneOf[getMatchingOption(undefined, schema.oneOf, rootSchema)];
+      schema.oneOf[
+        getMatchingOption(
+          JSON.stringify(formData) !== "{}" ? formData : undefined,
+          schema.oneOf,
+          rootSchema
+        )
+      ];
   } else if ("anyOf" in schema) {
     schema =
-      schema.anyOf[getMatchingOption(undefined, schema.anyOf, rootSchema)];
+      schema.anyOf[
+        getMatchingOption(
+          JSON.stringify(formData) !== "{}" ? formData : undefined,
```

Comment:
We should use the lodash function `isEmpty` instead, as it will make calculation faster (it could be slow to run JSON.stringify on the entire formData).

Reply:
Replaced with lodash `isEmpty`